esmvaltool/cmorizers/data/formatters/datasets/grace.py

```diff
@@ -39,7 +39,7 @@
 from dateutil import relativedelta
```

Comment:
This script is currently not working because of the call to the cmorization function.

Reply:
I want to test, but the download instructions for GRACE are not clear to me. It is probably because NASA is changing their data infrastructure. Can you take a look?